ariadne_codegen/client_generators/dependencies/async_base_client.py

```diff
@@ -164,11 +166,17 @@ async def execute_ws(
         ) as websocket:
             await self._send_connection_init(websocket)
             # wait for connection_ack from server
-            await self._handle_ws_message(
-                await websocket.recv(),
-                websocket,
-                expected_type=GraphQLTransportWSMessageType.CONNECTION_ACK,
-            )
+            # if ping is received, send pong and continue to wait for connection_ack
```

Comment:
Please make comment explain that some servers can send ping before `connection_ack`.